deps/jerry/jerry-core/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ set(FEATURE_DEBUGGER           ON       CACHE BOOL   "Enable JerryScript debugge
 set(FEATURE_ERROR_MESSAGES     ON       CACHE BOOL   "Enable error messages?")
 set(FEATURE_EXTERNAL_CONTEXT   OFF      CACHE BOOL   "Enable external context?")
 set(FEATURE_JS_PARSER          ON       CACHE BOOL   "Enable js-parser?")
-set(FEATURE_MEM_STATS          OFF      CACHE BOOL   "Enable memory statistics?")
+set(FEATURE_MEM_STATS          ON      CACHE BOOL   "Enable memory statistics?")
```

Comment:
We might be disable this in general, and enable it by compiling argument?

Reply:
should we enable this feature to  keep consistent with node.js.

Reply:
Let's disable it for now, if not too much performance cost, we could enable it then.